sanic/response/types.py

```diff
@@ -521,7 +521,7 @@ def __init__(
         content_type: Optional[str] = None,
     ):
         if not isinstance(headers, Header):
-            headers = Header(headers)
+            headers = Header(headers) # type: ignore
```

Comment:
Same.

Reply:
`sanic\response\types.py:524: error: Argument 1 to "Header" has incompatible type "Optional[Dict[str, str]]"; expected "Union[Mapping[str, Any], Mapping[istr, Any], Dict[str, Any], Dict[istr, Any], MultiMapping[Any], Iterable[Tuple[str, Any]], Iterable[Tuple[istr, Any]]]"`